eth/db/chain.py

```diff
@@ -118,19 +108,16 @@ def _set_as_canonical_chain_head(
                 old_header = cls._get_block_header_by_hash(db, old_hash)
                 old_canonical_headers.append(old_header)
                 try:
-                    for transaction_hash in cls._get_block_transaction_hashes(db, old_header):
+                    transaction_hashes = cls._get_block_transaction_hashes(db, old_header)
+                    for transaction_hash in transaction_hashes:
```

Comment:
This split was just to do fewer things on one line, right? (I'm fine with it, just wondering if there was something subtle I was missing)

Reply:
Nope, no subtle reason, the change was to get us under the 90-character line width limit